app/models.py

```diff
@@ -171,6 +171,28 @@ class Person(SearchableMixin, db.Model):
     education = db.Column(db.String)
     publications = db.Column(db.String)
 
+    def redact_data(self, fields):
+        if not fields["image"]:
+            self.image = None
+        if not fields["email"]:
+            self.email = None
+        if not fields["room"]:
+            self.entryway = None
+            self.floor = None
+            self.suite = None
+            self.room = None
+        if not fields["phone"]:
+            self.phone = None
+        if not fields["address"]:
+            self.address = None
```

Comment:
hmmmm. I think we should possibly label this as hometown instead of address.

Reply:
I tried to follow the Yale Face Book labels as closely as possible, and they use "Address" as their labels for their radio buttons. What do you think?

Reply:
Eh, I guess that's fair